user_sync/connector/directory_okta.py

```diff
@@ -227,9 +227,10 @@ def convert_user(self, record, extended_attributes):
             source_attributes['lastName'] = None
 
         if profile.countryCode:
-            source_attributes['countryCode'] = user['country'] = profile.countryCode
+            source_attributes['countryCode'] = profile.countryCode
+            user['country'] = profile.countryCode.upper()
         else:
-            source_attributes['countryCode'] = None
+            source_attributes['countryCode'] = user['country'] = None
```

Comment:
No, the field on the user object is not supposed to be set in this case; only the source object.  This line should stay the same.